### PR TITLE
Upgraded to bernard 1.0.0-alpha3

### PR DESCRIPTION
**Pull request by awartoft**

### CODE REVIEW

**Comment by awartoft** [comment]
@Hotas2k4 @noggan PLZ review

File: README.md [hunk 1]
```diff
@@ -3,14 +3,24 @@ ZF-Bernard
...
 
 ## Installing
 
 We support installation via composer
 
 ```composer require interactive-solutions/zf-bernard```
 
+## Normalizers
+
+You can add any normalizer that you which to use by adding it to the `BernardOptions::enabledNormalizers`
```
Comment (Hotas2k) [inline README.md]: wish to use

File: src/Factory/PersistentFactoryFactory.php [hunk 2]
```diff
@@ -12,21 +12,30 @@
...
 use Bernard\QueueFactory\PersistentFactory;
 use Bernard\Serializer;
-use Zend\ServiceManager\FactoryInterface;
-use Zend\ServiceManager\ServiceLocatorInterface;
+use InteractiveSolutions\Bernard\BernardOptions;
+use Interop\Container\ContainerInterface;
+use Normalt\Normalizer\AggregateNormalizer;
 
-class PersistentFactoryFactory implements FactoryInterface
+class PersistentFactoryFactory
```
Comment (noggan) [inline src/Factory/PersistentFactoryFactory.php:19]: final

File: src/Factory/ConsumerFactory.php [hunk 3]
```diff
@@ -10,28 +10,27 @@
...
 
 use Bernard\Consumer;
-use Bernard\Middleware\MiddlewareBuilder;
+use InteractiveSolutions\Bernard\EventDispatcherInterface;
 use InteractiveSolutions\Bernard\Router\PluginManagerRouter;
-use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
-class ConsumerFactory implements FactoryInterface
+class ConsumerFactory
```
Comment (noggan) [inline src/Factory/ConsumerFactory.php:17]: final

File: src/Factory/ProducerFactory.php [hunk 4]
```diff
@@ -9,45 +9,25 @@
...
-use InteractiveSolutions\Bernard\Middleware\Producer;
-use Symfony\Component\DependencyInjection\ContainerInterface;
-use Zend\ServiceManager\FactoryInterface;
-use Zend\ServiceManager\ServiceLocatorInterface;
+use InteractiveSolutions\Bernard\EventDispatcherInterface;
+use InteractiveSolutions\Bernard\Producer;
+use Interop\Container\ContainerInterface;
 
-class ProducerFactory implements FactoryInterface
+class ProducerFactory
```
Comment (noggan) [inline src/Factory/ProducerFactory.php:18]: final

File: src/Factory/ProducerFactory.php [hunk 5]
```diff
@@ -9,45 +9,25 @@
...
 {
-    /**
-     * {@inheritdoc}
-     */
-    public function createService(ServiceLocatorInterface $serviceLocator):Producer
```
Comment (noggan) [inline src/Factory/ProducerFactory.php]: missing whitespace after `:`

**Pull request merged**